### PR TITLE
Adoption: Add Neutron DHCP Agent to standlone deployment

### PR DESCRIPTION
**Pull request by umago**
This patch adds the Neutron DHCP Agent to the standalone deployment to be used for adoption.

The patch also adds a configuration to allow disabling it if needed, by default it's enabled by default as it will be used in the adoption CI.

Related: #[OSPRH-2435](https://issues.redhat.com//browse/OSPRH-2435)

### CODE REVIEW

**Review by slawqo — COMMENTED** [review]
LGTM for me, thx

File: devsetup/scripts/standalone.sh [hunk 1]
```diff
@@ -111,6 +112,15 @@ parameter_defaults:
     NeutronPhysicalDevMappings: "dummy_sriov_net:dummy-dev"
 __EOF__
 
+cat >\$HOME/dhcp_agent_template.yaml <<__EOF__
+parameter_defaults:
+  NeutronMechanismDrivers: ['ovn']
```
[the remark this answers is not in the document]
Comment (umago) [reply devsetup/scripts/standalone.sh:117]: Good point, perhaps it's not needed. I will update the PR

**Comment by fao89** [comment]
recheck

**Review by slawqo — APPROVED** [review]

**Pull request merged**